packages/internal/xrpc-utils/src/accept.ts

```diff
@@ -0,0 +1,143 @@
+import { ResponseType } from '@atproto/xrpc'
```

Comment:
I am assuming this is a direct port from packages/pds/src/pipethrough.ts, so not reviewing in detail.

Reply:
Just a copy paste